### ISSUE TITLE
NVDA not reading grouping label in Chrome

### ISSUE THREAD

**Issue by terencet** [nvaccess/nvda#7205]
### Steps to reproduce:

To demonstrate the issue I have setup a page hosted on dropbox.

1. Ensure NVDA is running
2. Load NVDA Speech Viewer
3. Open Chrome, go to https://dl.dropboxusercontent.com/s/4em2b05sipkwlip/sample.html

Also here is the link to the sample HTML hosted in gist
https://gist.github.com/terencet/9626d90bbb4ecee8265868148b5d94e6

The URL display 2 groups of buttons and each group has a label. Resume and Cover Letter

### Expected behaviour:  
1. After the page load
2. Press tab to go to the first control
3. NVDA/Speech Viewer should display/read 

> Upload Some Files
>Resume* grouping
> list with 3 items
> Upload file
> button

### Actual behaviour:
1. After the page load
2. Press tab to go to the first control
3. NVDA/Speech Viewer displayed/read 

> Upload Some Files
> list with 3 items
> Upload file
> button

### System configuration:
NVDA version: 2017.1
NVDA Installed

Other information:
Windows version:
Windows 8.1 Pro
Chrome Version 58.0.3029.110 (64-bit)

The only way to force NVDA to read the group label is to add a textbox control just above each grouping.

### Other questions:
Does the issue still occur after restarting your PC?
Yes

Have you tried any other versions of NVDA?
No

**Comment by feerrenrut** [comment]
I can reproduce the behaviour you are describing in Chrome. However when I use an example like the one on the [w3 HTML5 spec](https://www.w3.org/TR/html5/forms.html#the-fieldset-element) the behaviour seemed fine.

I tried to create a [minimal example of this](https://gist.github.com/feerrenrut/64982e55d53f5a272e278e02a8edb67a). You will notice that this example has an input field outside of the fieldset. If you uncomment this line, it becomes much easier to test if the grouping is working correctly. I tried this with your example and found that the grouping is reported. Essentially, if you first focus an element outside the grouping, then tab to enter the grouping, then the grouping is reported.

Personally I find this behaviour (not reading the grouping when tabing into it as the first field in the document) a little confusing, @jcsteh I would like to get your opinion here.

Other notes:
* "grouping" is not ever read in browse mode, only `Resume*  list with 3 items`. When tabing  (in focus mode) from the input field I added outside the fieldset element the following is read `Resume*  grouping   required  list  Upload file`
* When tabing from one grouping to the next, the new grouping is not read. Again if I add a input field between these two lists the names of the second group is read.
